lib/internal/util.js

```diff
@@ -278,3 +278,14 @@ promisify.custom = kCustomPromisifiedSymbol;
 
 exports.promisify = promisify;
 exports.customPromisifyArgs = kCustomPromisifyArgsSymbol;
+
+exports.intrinsic = {
+  FunctionApply: Function.prototype.apply,
+  FunctionCall: Function.prototype.call,
+  ObjectAssign: Object.assign,
+  ObjectSetPrototypeOf: Object.setPrototypeOf,
+  Object: Object,
+  Array: Array,
+  ObjectPrototype: Object.assign({}, Object.prototype),
```

Comment:
I'd rather save the prototype methods individually, and set ObjectPrototype to the initial `Object.prototype`.